Billing.API/Controllers/InvoiceController.cs

```diff
@@ -20,10 +22,10 @@ public InvoiceController(ILogger<InvoiceController> logger, IInvoiceService invo
             _invoiceService = invoiceService;
         }
 
-        [HttpGet("/invoices/{clientId:length(15)}")]
-        public async Task<IActionResult> GetInvoices([FromRoute] string clientId)
+        [HttpGet("/accounts/{origin}/{clientId:length(15)}/invoices/")]
+        public async Task<IActionResult> GetInvoices([FromRoute] ClientOrigin origin = ClientOrigin.Doppler, [FromRoute] string clientId = "", [FromQuery] DateTime? from = null, [FromQuery] DateTime? to = null)
```

Comment:
If I am not wrong, ClientOrigin and ClientId default values are not necessary. 

And, if you are not taking into account date filter yet, you can remove this

```suggestion
        public async Task<IActionResult> GetInvoices([FromRoute] ClientOrigin origin, [FromRoute] string clientId)
```